src/test/java/unit/CloneWorkspace.java

```diff
@@ -182,13 +182,16 @@ public void cloneWorkspace(TestInfo testInfo) throws Exception {
             "clone",
             "--new-id=cloned_id" + RANDOM.nextInt(Integer.MAX_VALUE),
             "--name=cloned_workspace",
-            "--description=A clone.");
+            "--description=A clone.",
+            "--property=key1=value1,key2=value2");
```

Comment:
So this usage makes me think maybe we should name the option `--properties`. When it's used multiple times, the singular makes sense.